topics/metagenomics/tutorials/metagenomics-assembly/tutorial.md

```diff
@@ -0,0 +1,286 @@
+>
+> 1. Create new history
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. At the top of the Tools panel (on the left), click “Upload Data”
+> ![Upload data](../../images/metagenomics-assembly/upload-data.png){:width="40%"}
+> This brings up a box:
+> ![Download data](../../images/metagenomics-assembly/download-data.png){:width="80%"}
+> 
+> 3. Click Paste/Fetch data and paste in the following URLs in the box that appears.
+>  ```
+> https://zenodo.org/record/6958893/files/ERR2231567_Read_1.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231567_Read_2.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231568_Read_1.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231568_Read_2.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231569_Read_1.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231569_Read_2.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231570_Read_1.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231570_Read_2.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231571_Read_1.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231571_Read_2.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231572_Read_1.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231572_Read_2.fastqsanger.gz
+>  ```
+> 4. Click Start, and then Close
+> 5. When they are ready, rename the datasets
+{: .hands_on}
+
+### Assembly
+Co-assembly of short reads will be the focus of this tutorial. Later in this tutorial, we will also cover individual assembly. 
+To accomplish our objectives, we will use the MetaSPAdes assembly tool. MetaSPADES is an NGS assembler designed specifically for large and complex metagenomics datasets. MetaSPAdes is part of the SPAdes toolkit, which has several assembly pipelines.
+
+There are three challenges to metagenomics assembly: 1) differences in genome coverage between samples, resulting from differences in abundance, 2) the fact that different species often share conserved regions ({%cite kececioglu2001%}), and 3) the presence of multiple strains of a single species ({%cite miller2010%}). Since SPAdes handles non-uniform coverage, it is useful for assembling simple communities, but metaSPAdes also handles other problems, allowing it to assemble complex communities' metagenomes.
+
+#### Co-assembly
+
+> ### {% icon hands_on %} Hands-on: Assembly with MetaSPAdes
+>
+> > ### {% icon comment %} Comment
+> >
+> > In the next steps we use **metaSPAdes** assembler. **MetaSPAdes** is a versatile metagenomic assembler ({%cite nurk2017%}). As input for metaspades it can accept short reads. However, there is an option to use additionally long reads besides short reads to produce hybrid input.
+> {: .comment} 
+>
+> 1. Name a new history with the name “Assembly with MetaSPAdes”
+>
+> 2. Run **MetaSPAdes**{% icon tool %} with parameters:
```

Comment:
Can you add for all tools their link like explained: https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links? Thanks